web_notify/tests/test_res_users.py

```diff
@@ -17,12 +18,18 @@ def test_notify_info(self):
         ]
         existing = bus_bus.search(domain)
         self.env.user.notify_info(
-            message='message', title='title', sticky=True)
+            message='message', title='title', sticky=True,
+            show_reload=True, foo="bar"
+        )
         news = bus_bus.search(domain) - existing
         self.assertEqual(1, len(news))
-        self.assertEqual(
-            '{"message":"message","sticky":true,"title":"title"}',
-            news.message)
+        expected = ('{"message":"message","sticky":true,"title":"title",'
```

Comment:
one attribute per line makes it easier to read

Reply:
this is a dict in a string in json format for test purpose, I think it's okay this way as it is an expected output.